src/tls.c

```diff
@@ -168,6 +170,94 @@ int ssl_init()
            ERR_error_string(ERR_get_error(), NULL));
     ERR_free_strings();
   }
+  /* Let advanced users specify the list of allowed ssl protocols */
+  #define EGG_SSLv2   1 << 0
```

Comment:
paranthesis around definition please (1 << 0), in case it's moved and used in odd places

Reply:
done